contents/handbook/engineering/clickhouse/overview.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: About
+---
+
+Welcome to PostHog's ClickHouse manual.
+
+## About this manual
+
+PostHog uses ClickHouse to power our data analytics tooling and we've learned a lot about it over the years. The goal
+of this manual is to share that knowledge externally and raise the average level of ClickHouse understanding for
+people starting work with ClickHouse.
+
+If you have extensive ClickHouse experience, and want to contribute thoughts or tips of your own, please do!
+
+Consider this manual a companion to other great resources out there:
+- https://clickhouse.com/docs/en/
+- https://kb.altinity.com/
+- https://tinybird.co/clickhouse/knowledge-base/
+
+## Why ClickHouse
```

Comment:
@fuziontech Shamelessly stole this from your past blog post: https://posthog.com/blog/how-we-turned-clickhouse-into-our-eventmansion

I think this is OK but if you have better content for this feel free to edit this as you'd like :)